.github/workflows/build_and_test.yml

```diff
@@ -4820,14 +4820,14 @@ jobs:
       run: |
         cd OpenDDS
         call setenv.cmd
-        cd tests\DCPS\UnitTests
+        cd tests\unit-tests
         perl %ACE_ROOT%\bin\mwc.pl -type vs2019 -static -features no_rapidjson=1 -features ipv6=1 -features ssl=1 -features no_cxx11=0
     - name: make unit tests
       shell: cmd
       run: |
         cd OpenDDS
         call setenv.cmd
-        cd tests\DCPS\UnitTests
+        cd tests\unit-tests
         msbuild -p:Configuration=Debug,Platform=x64 -m UnitTests.sln
```

Comment:
@ClaytonCalabrese @mitza-oci There are multiple mpc files in `tests/unit-tests`.  How should I update the line that contains `UnitTests.sln`?

Reply:
If there are multiple .sln files from the previous step, I suppose we just iterate over them? I'm not on a windows machine to test, but maybe something like:
`for %f in (.\*.sln) do msbuild -p:Configuration=Debug,Platform=x64 -m %f`

It's possible that passing *.sln to msbuild might work as well...

Reply:
Why would there be multiple workspaces?